### DISCUSSION TITLE
Trouble installing rsvg-convert

### DISCUSSION THREAD

**Discussion by vogtr19** [quarto-dev/quarto-cli#10547] [Q&A] [answered]
### Description

Build information: Running Quarto 1.5.56, RStudio 2024.04.2 Build 764 on Windows Dell XPS17 laptop
Background: working on a Quarto book project and trying to render a pdf from content including a .svg file

The original error message received prompting me to try to install rsvg-convert was during 'quarto render --to pdf': "Could not convert a SVG to a PDF for output.  Please ensure that rsvg-convert is available on the path". A google search for 'rsvg-convert Quarto' included the following URL:
https://quarto.org/docs/prerelease/1.3/pdf.html
This URL indicates to install chocolatey for Windows using this in command line: choco install rsvg-convert

When I tried running this in Rstudio Console and Terminal windows, as well as Windows Command Prompt (it wasn't clear which prompt to use in my situation) I got the following error messages: 
Rstudio Terminal and Windows Command Prompt:  'choco' is not recognized as an internal or external command, operable program or batch file.
Rstudio Console: Error: unexpected symbol in "choco install"

Thanks for any help you can provide.

**Comment by cscheid** [comment]
You need to install Chocolatey before you can use `choco`, the Chocolatey command.

**Reply by vogtr19** [reply]
Thanks!  I'm a bit embarrassed I didn't try that with the name spelled correctly.  Unfortunately, chocolatey is not compatible with my installed v4.4.1 of R.  I don't really want to use an earlier version just to run this.  I may need to find a workaround utility or something that can convert my .svg file to .pdf or other file type quarto can work with.

**Reply by mcanouil** [reply]
Chocolatey is independent and unrelated to R.
Are you talking about rsvg?

**Reply by vogtr19** [reply]
Sorry, I had to look back to refresh my memory.  Upon trying install.packages("chocolatey"), I got the message: 
WARNING: Rtools is required to build R packages but is not currently installed. Please download and install the appropriate version of Rtools before proceeding:

https://cran.rstudio.com/bin/windows/Rtools/
Installing package into ‘C:/Users/other/AppData/Local/R/win-library/4.4’
(as ‘lib’ is unspecified)
Warning in install.packages :
  package ‘chocolatey’ is not available for this version of R

When I try install.packages("Rtools") I get message:
WARNING: Rtools is required to build R packages but is not currently installed. Please download and install the appropriate version of Rtools before proceeding:

https://cran.rstudio.com/bin/windows/Rtools/
Installing package into ‘C:/Users/other/AppData/Local/R/win-library/4.4’
(as ‘lib’ is unspecified)
Warning in install.packages :
  package ‘Rtools’ is not available for this version of R

Having a hard time interpreting next steps here.  Thanks so much for the inputs so far and your patience.

Ryan

**Reply by mcanouil** [reply, marked as answer]
Chocolatey is not an R package, see <https://community.chocolatey.org>.

rsvg-convert is also not an R package, it's a system library.

**Reply by vogtr19** [reply]
Thanks, all!  quarto render --to pdf now working perfectly with the .svg file..
